public/app/plugins/panel/alertlist/UnifiedAlertList.tsx

```diff
@@ -62,61 +63,119 @@ export function UnifiedAlertList(props: PanelProps<UnifiedAlertListOptions>) {
+                </div>
+              </li>
+            ))}
+          {props.options.groupMode === GroupMode.Default && (
+            <ol className={styles.alertRuleList}>
+              {haveResults &&
+                rulesToDisplay.map((ruleWithLocation, index) => {
+                  const { rule, namespaceName, groupName } = ruleWithLocation;
+                  const firstActiveAt = getFirstActiveAt(rule);
+                  return (
+                    <li
+                      className={styles.alertRuleItem}
+                      key={`alert-${namespaceName}-${groupName}-${rule.name}-${index}`}
+                    >
+                      <div className={stateStyle.icon}>
+                        <Icon
+                          name={alertDef.getStateDisplayModel(rule.state).iconClass as IconName}
+                          className={stateStyle[alertStateToState[rule.state]]}
+                          size={'lg'}
+                        />
+                      </div>
+                      <div>
+                        <div className={styles.instanceDetails}>
+                          <div className={styles.alertName} title={rule.name}>
+                            {rule.name}
+                          </div>
+                          <div className={styles.alertDuration}>
+                            <span className={stateStyle[alertStateToState[rule.state]]}>
+                              {rule.state.toUpperCase()}
+                            </span>{' '}
+                            {firstActiveAt && rule.state !== PromAlertingRuleState.Inactive && (
+                              <>
+                                for{' '}
+                                <span>
+                                  {intervalToAbbreviatedDurationString({
+                                    start: firstActiveAt,
+                                    end: Date.now(),
+                                  })}
+                                </span>
+                              </>
+                            )}
+                          </div>
                         </div>
+                        <AlertInstances alerts={ruleWithLocation.rule.alerts ?? []} options={props.options} />
                       </div>
-                      <AlertInstances ruleWithLocation={ruleWithLocation} options={props.options} />
-                    </div>
-                  </li>
-                );
-              })}
-          </ol>
+                    </li>
+                  );
+                })}
+            </ol>
+          )}
         </section>
       </div>
     </CustomScrollbar>
   );
 }
 
+function createMapKey(groupBy: string[], labels: Record<string, string>): string {
+  return new URLSearchParams(groupBy.map((key) => [key, labels[key]])).toString();
+}
+
+function parseMapKey(key: string): Array<[string, string]> {
+  // @ts-ignore dom.iterable is not added to the @grafana/tsconfig but this works just fine
```

Comment:
Can we ask someone to add it? Or can we do it? I think I had a similar problem

Reply:
Yep, I got the ball rolling on this one and has now been merged in main https://github.com/grafana/grafana/pull/44582 so I will rebase.